core/quill.js

```diff
@@ -467,6 +469,27 @@ Quill.events = Emitter.events;
 Quill.sources = Emitter.sources;
 // eslint-disable-next-line no-undef
 Quill.version = typeof QUILL_VERSION === 'undefined' ? 'dev' : QUILL_VERSION;
+Quill.LIST_STYLE_KEY = 'mso-list-data';
+
+Quill.replaceStyleAttribute = (html) => {
+  const tagAttrsRegex = /(?:(<[a-z0-9]+\s*))([\s\S]*?)(>|\/>)/gi;
+
+  return html.replace(tagAttrsRegex, (allTagAttrs, tagStart, tagAttrs, tagEnd) => {
+    const contentWithoutStyle = tagAttrs.replace(/style(\s?)+=/gi, `${STYLE_ATTRIBUTE_KEY}=`);
+
+    return tagStart + contentWithoutStyle + tagEnd;
+  });
+};
+
+Quill.restoreStyleAttribute = (element) => {
+  element.querySelectorAll(`[${STYLE_ATTRIBUTE_KEY}]`).forEach((currentElement) => {
+    const attrValue = currentElement.getAttribute(STYLE_ATTRIBUTE_KEY);
+
+    currentElement.style = attrValue;
+    currentElement.setAttribute(Quill.LIST_STYLE_KEY, attrValue);
```

Comment:
I see that we set `Quill.LIST_STYLE_KEY` attr in the same time as usual `style`. So I don't get why we need it, why cannot just code in devextreme still look at `style`?

Reply:
[discussed personally]. It's necessary beucase EM specific styles are removed if we assign `style` at runtime